packages/design-system/src/styles/_base.scss

```diff
@@ -155,15 +143,14 @@ $breakpoints: (
 }
 
 @mixin heading-margins {
-  margin-bottom: 0.5em;
-  margin-top: 1.5em;
+  margin-block: 1.5em 0.5em;
```

Comment:
Since I was cleaning up CSS, I updated many of our layout rules to use [CSS logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties). In a nutshell, it offers the same functionality as before but now with added support for non-left-to-right language preferences.

Reply:
Do you think we'll need any documentation for this shift for our users?

Reply:
Our users shouldn't see a difference, so I don't think anything outside of our normal release notes would be warranted. Now, if we wanted to go a step further and update the name of our utility classes to match the logical property syntax, that would need documentation but I don't see a need to update util class names just yet.